### PR TITLE
(GH-36) Add rules for non-player companies

### LINKED ISSUE #36: Add Automated Procedures
Add a section to the rules for automated procedures, explaining their general use (referring to default procedures/etc where necessary).

### PR DESCRIPTION
**Pull request by michaeltlombardi**
This change introduces guidance for running non-player Companies based on a prior blog post. It includes six basic personalities which may serve as guidance for introducing new personalities.

- Resolves #36

### CODE REVIEW

File: content/games/factions/routines.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+1. **Arcane:** Prefer to use and acquire magic over mundanity.
+1. **Bold:** Chase objectives and move without concern for enemy action.
+1. **Cautious:** Steer clear of enemies unless odds are favorable.
+1. **Ferocious:** Destroy the enemy whenever possible.
+1. **Guarded:** Avoid attacking directly, prefer to hold and whittle.
+1. **Incompetent:** Follow default/random orders.
+
+## Traits
+
+Where a trait would be triggered by player choice, roll a d6: on a 4+, the trait is triggered.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:24]:
```suggestion
Where a trait would be triggered by player choice, roll 1d6: on 4+, the trait triggers.
```

File: content/games/factions/routines.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+Guarded Companies activate as normal but may not activate to Attack an enemy Group unless a friendly
+Group is within 12". Groups not within 12" of a friendly Group must move towards the nearest
+friendly Group or defensible position.
+
+## Eccentrics
+
+- Groups in Bold and Ferocious Companies with the Defiant trait must always test to activate it in
+  response to being Attacked.
+- Groups in Cautious and Guarded Companies with the Elusive trait must always test to activate it in
+  response to being Attacked.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:135]:
```suggestion
- Groups in Bold and Ferocious Companies with the Defiant trait must always test to activate it when
  Attacked.
- Groups in Cautious and Guarded Companies with the Elusive trait must always test to activate when
  Attacked.
```

File: content/games/factions/routines.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+- Groups with the Casting order may not activate to Cast the same spell twice in a row and cast in
+  one of two orders:
+  1. If the Company has the Arcane Personality, cast from highest activation difficulty to lowest if
+     there is a valid target, otherwise cast the next highest difficulty spell. May not recast the
+     same spell until all known spells have been attempted at least once.
+  1. Otherwise, cast randomly against nearest valid target.
+- Groups with Foe or Bane traits _must_ Move towards and attempt to Attack the nearest specified
+  Kind of Group if within 18".
+- Groups with Self-Destruct trait _must_ attempt to activate it once they are under half their
+  starting R.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:146]:
```suggestion
- Groups with Foe or Bane traits must Move towards and attempt to Attack the nearest specified
  Kind of Group if within 18".
- Groups with Self-Destruct trait must attempt Self-Destruct once they are Bloodied.
```

File: content/games/factions/routines.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+Cautious Companies activate as normal but may not activate to Attack an enemy Group whose R is equal
+to or greater than their own. Groups more than 6" away from a friendly Group must Move towards one
+if possible.
+
+### Ferocious Group Activations
+
+Ferocious Companies must activate the Group nearest an enemy first, then the next closest Group, and
+so on. Groups capable of Attacking or Shooting must do so, preferring whichever activation order has
+a better hit chance. Groups not within range of an objective or enemy must Move towards the nearest
+enemy Group.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:122]:
```suggestion
Ferocious Companies must activate the Group nearest an enemy that has not already activated. Groups
capable of Attacking or Shooting must do so, preferring whichever activation order has a better hit
chance. Groups not within range of an objective or enemy must Move towards the nearest enemy Group.
```

File: content/games/factions/routines.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+Bold Companies must activate the Group nearest an enemy or objective first, then the next closest
+Group, and so on. Groups capable of seizing an objective or attacking an enemy must do so,
+preferring objectives over attacking.
+
+### Cautious Group Activations
+
+Cautious Companies activate as normal but may not activate to Attack an enemy Group whose R is equal
+to or greater than their own. Groups more than 6" away from a friendly Group must Move towards one
+if possible.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:115]:
```suggestion
Cautious Companies may not Attack an enemy Group whose R is equal to or greater than their own.
Groups more than 6" away from a friendly Group must Move towards one if possible.
```

File: content/games/factions/routines.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+Arcane Companies must activate any Groups with the Casting, Apprentice, or Summoner traits before
+others, from highest to lowest point value. Groups capable of the Casting or Summoning order _must_
+use that order. If capable of both, alternate between them each time.
+
+### Bold Group Activations
+
+Bold Companies must activate the Group nearest an enemy or objective first, then the next closest
+Group, and so on. Groups capable of seizing an objective or attacking an enemy must do so,
+preferring objectives over attacking.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:109]:
```suggestion
Bold Companies must activate the Group nearest an enemy or objective that has not already activated.
Groups capable of seizing an objective or attacking an enemy must do so, preferring objectives over
attacking.
```

File: content/games/factions/routines.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+   cannot perform the specified action, default to the next lower-order action until they can.
+   - If moving, roll 1d6 and: move away from nearest enemy (1--2), move towards nearest enemy
+     (3--4), move towards nearest objective (5--6).
+1. Resolve each activation as normal.
+
+### Arcane Group Activations
+
+Arcane Companies must activate any Groups with the Casting, Apprentice, or Summoner traits before
+others, from highest to lowest point value. Groups capable of the Casting or Summoning order _must_
+use that order. If capable of both, alternate between them each time.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:103]:
```suggestion
Arcane Companies must activate any Groups with the Casting or Summoner order before others, from
highest to lowest point value. Groups capable of the Casting or Summoning order _must_ use that
order. If capable of both, alternate between them each time.
```

File: content/games/factions/routines.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+Groups.
+
+When activating individual Groups, unless specified, they follow this process:
+
+1. Must activate the Captain's Group first, then the closest Group to the Captain's Group which has
+   not yet activated.
+1. Roll 1d6 to determine the activation: Move (1--2), Attack (3--4), Shoot (5--6). If the group
+   cannot perform the specified action, default to the next lower-order action until they can.
+   - If moving, roll 1d6 and: move away from nearest enemy (1--2), move towards nearest enemy
+     (3--4), move towards nearest objective (5--6).
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:96]:
```suggestion
   - If Moving, roll 1d6 and: Move away from nearest enemy (1--2), Move towards nearest enemy
     (3--4), Move towards nearest objective (5--6).
```

File: content/games/factions/routines.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+weight: 50
+---
+
+These procedures and guidelines apply to Companies not controlled by a player. Unless otherwise
+specified, Routine Companies follow the same rules as other companies. Where decisions are unclear,
+use judgement.
+
+## Personalities
+
+Routine companies have a Personality. Roll a d6 or choose for each Routine Company:
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:13]:
```suggestion
Routine companies have a Personality. Roll 1d6 or choose for each Routine Company:
```

File: content/games/factions/routines.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+
+1. **Arcane:** Prefer to use and acquire magic over mundanity.
+1. **Bold:** Chase objectives and move without concern for enemy action.
+1. **Cautious:** Steer clear of enemies unless odds are favorable.
+1. **Ferocious:** Destroy the enemy whenever possible.
+1. **Guarded:** Avoid attacking directly, prefer to hold and whittle.
+1. **Incompetent:** Follow default/random orders.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:20]:
```suggestion
1. **Incompetent:** Follow default/random orders.
1. **Arcane:** Prefer to use and acquire magic over mundanity.
1. **Bold:** Chase objectives and move without concern for enemy action.
1. **Cautious:** Steer clear of enemies unless odds are favorable.
1. **Ferocious:** Destroy the enemy whenever possible.
1. **Guarded:** Avoid attacking directly, prefer to hold and whittle.
```

File: content/games/factions/routines.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+     zone.
+1. For each remaining Group, roll 1d6:
+   - Select the Group from the right/left of the line if the result is even/odd respectively.
+   - If the result is _lower_ than the previous roll, deploy in an adjacent quadrant.
+   - If the result is _equal_ to the previous roll, deploy in the same quadrant.
+   - If the result is _higher_ than the previous roll, deploy in the opposite quadrant.
+1. Depending on the Company's Personality, these deployments may need to be adjusted, either
+   swapping Groups or moving them within their deployment quadrant:
+   - Arcane Companies must deploy their magic using Groups within 6" of a friendly group if
+     possible.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:60]:
```suggestion
   - Arcane Companies must deploy Groups with the Casting or Summoning order within 6" of a friendly
     group if possible.
```

File: content/games/factions/routines.md [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+this process:
+
+1. Line up the Routine Company's Groups from lowest to highest point value along a table edge.
+1. Roll 1d6 to deploy the first Group:
+   - If the result is _even_, deploy the group furthest right in the line to the nearest deployment
+     zone.
+   - If the result is _odd_, deploy the group furthest left in the line to the furthest deployment
+     zone.
+1. For each remaining Group, roll 1d6:
+   - Select the Group from the right/left of the line if the result is even/odd respectively.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:53]:
```suggestion
   - Select the Group from the left/right of the line if the result is odd/even respectively.
```

File: content/games/factions/routines.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+
+Ferocious Companies must activate the Group nearest an enemy that has not already activated. Groups
+capable of Attacking or Shooting must do so, preferring whichever activation order has a better hit
+chance. Groups not within range of an objective or enemy must Move towards the nearest enemy Group.
+
+### Guarded Group Activations
+
+Guarded Companies activate as normal but may not activate to Attack an enemy Group unless a friendly
+Group is within 12". Groups not within 12" of a friendly Group must move towards the nearest
+friendly Group or defensible position.
```
Comment (s-jared) [inline content/games/factions/routines.md:126]:
```suggestion
Guarded Companies may not activate to Attack an enemy Group unless there is a friendly
Group within 12". Groups not within 12" of a friendly Group must move toward the nearest
friendly Group if possible, or toward the nearest defensible position otherwise.
```

File: content/games/factions/routines.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+- Spells are chosen randomly unless specified beforehand.
+- Groups with the Casting order may not activate to Cast the same spell twice in a row and cast in
+  one of two orders:
+  1. If the Company has the Arcane Personality, cast from highest activation difficulty to lowest if
+     there is a valid target, otherwise cast the next highest difficulty spell. May not recast the
+     same spell until all known spells have been attempted at least once.
+  1. Otherwise, cast randomly against nearest valid target.
+- Groups with Foe or Bane traits must Move towards and attempt to Attack the nearest specified
+  Kind of Group if within 18".
+- Groups with Self-Destruct trait must attempt Self-Destruct once they are Bloodied.
```
Comment (s-jared) [inline content/games/factions/routines.md:143]:
```suggestion
- Groups in Bold and Ferocious Companies with the Defiant trait must always test to activate that trait when possible.
- Groups in Cautious and Guarded Companies with the Elusive trait must always test to activate when
  possible.
- Choose spells randomly unless specified otherwise.
- Groups with the Casting order may not activate to Cast the same spell twice in a row. If the Company has the Arcane Personality, cast from highest activation difficulty to lowest if
     there is a valid target, otherwise cast the next highest difficulty spell. May not recast the
     same spell until all known spells have been attempted at least once. Otherwise, cast randomly against the nearest valid target.
- Groups with Foe or Bane traits must Move towards and attempt to Attack the nearest specified
  type of Group if within 18" of one.
- Groups with Self-Destruct trait must attempt Self-Destruct once they are Bloodied.
```

Comment (s-jared) [reply content/games/factions/routines.md:143]: i really fubared the line breaks on this one, sorry

we need to talk about this stuff with how the Arcane Personality activates. i don't *quite* understand what you're going for. can you ELI5 it to me?

**Review by s-jared — COMMENTED** [review]
first pass, this is good stuff. im impressed with how straightforward it is

File: content/games/factions/routines.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+   not yet activated.
+1. Roll 1d6 to determine the activation: Move (1--2), Attack (3--4), Shoot (5--6). If the group
+   cannot perform the specified action, default to the next lower-order action until they can.
+   - If Moving, roll 1d6 and: Move away from nearest enemy (1--2), Move towards nearest enemy
+     (3--4), Move towards nearest objective (5--6).
+1. Resolve each activation as normal.
+
+### Arcane Group Activations
+
+Arcane Companies must activate any Groups with the Casting or Summoner order before others, from
```
[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:101]: I had this as `order` here to account for the Apprentice/Caster traits, both of which have the `Cast` order but I see I fucked up and said `Casting or Summoner` instead of `Cast or Summon`. I'm fine going either to the orders or the traits.

File: content/games/factions/routines.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+### Cautious Group Activations
+
+Cautious Companies may not Attack an enemy Group whose R is equal to or greater than their own.
+Groups more than 6" away from a friendly Group must Move towards one if possible.
+
+### Ferocious Group Activations
+
+Ferocious Companies must activate the Group nearest an enemy that has not already activated. Groups
+capable of Attacking or Shooting must do so, preferring whichever activation order has a better hit
+chance. Groups not within range of an objective or enemy must Move towards the nearest enemy Group.
```
[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:120]: Going to add `towards` to the "are you british? is the book british?" lol

File: content/games/factions/routines.md (hunk 14 above)
Comment (michaeltlombardi) [reply content/games/factions/routines.md:143]: Sure, so what I _meant_ to do here is:

1. Keep groups from spamming the same spell over and over
2. Have the arcane groups cycle from their best-to-worst spells where possible

File: content/games/factions/routines.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+## Traits
+
+Where a trait would be triggered by player choice, roll 1d6: on 4+, the trait triggers.
+
+## Setup
+
+### Initiative
+
+Incompetent Companies automatically forfeit their initiative roll, counting as a 1 for all purposes.
+When a Routine Company wins the initiative roll-off:
```
[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:31]: 🤔 I think we should go with option 2 and retain the multiplayer stuff here. 

I'm chewing over the idea that all companies roll for initiative and add their results to see which side wins. This would favor the side that has more companies pretty heavily, which makes some sense to me (more captains means more skilled people prepared to seize initiative).

I think the rest of the text doesn't account well for n-sided skirmishes (3+ sides). I think nearly everything is written with an attacker/defender dichotomy. This might not be a bad time to address that. I'm leaning towards explicit support but open to leaving that to the reader to figure out.

[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:31]: This would also mean clarifying which Company on a side goes first, which I think is also ambiguous right now.

File: content/games/factions/routines.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+
+- Bold and Ferocious Companies prefer to be on the offensive.
+- Cautious and Guarded Companies prefer to be on the defensive.
+- Arcane Companies prefer whichever role is more eldritch (e.g. the ritualists in the Ritual
+  scenario).
+
+If the Routine Company's Captain has the Shrewd trait, they use it towards their preference.
```
[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:38]: 👍 Agreed, incorporating this change.

File: content/games/factions/routines.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (s-jared) [reply content/games/factions/routines.md:31]: Okay so here's where I'm sitting with this, atm. I'm slightly wary of getting too "comprehensive" in the core rules. You're always at risk of beginning a recipe for bread with instructions for hybridizing your own strain of wheat. I'm into the idea of providing some kind of support, without getting to hysterical about it. The two options I like best are:

1. Just adding a little line/paragraph somewhere in the core rules like "Yo this is structured for attacker/defender, and assumes one company per side. We've tried to keep it a little open, but you'll have to do some legwork. I believe in you."
2. Add a little section to the core rules with some more explicit idea of how to handle the outside cases: What if >2 sides? What if >1 company on a side? This would be its own entire section, at the end of the thing, out of the way. It *might* involve tightening up the core to actually sit more squarely into "This is for two sides with one Company per side," but that's a bridge we can cross when we come to it.

Doing one of those things in the core rules would then give us some guidance as to how to handle things here in the auto rules. We'd basically mirror that same ethos from there to here, either keeping things as "This is attacker/defender and 1 Company each, figure it out if you want it different" or "That same thing, but here's a small section to account for n sides and n companies per side."

File: content/games/factions/routines.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (s-jared) [reply content/games/factions/routines.md:99]: Leaning on orders is better, actually. Good call.

File: content/games/factions/routines.md (hunk 14 above)
Comment (s-jared) [reply content/games/factions/routines.md:141]: okay so i fussed with the arcane personality stuff a bit. its... *a lot* to get in there, and it needs to be super compressed obvs, so tell me 1) if it reads, and 2) if it actually results in what you were going for. i *think* it does, but god knows

Comment (michaeltlombardi) [reply content/games/factions/routines.md:141]: Scans well enough to me and is as short as I think we can make it without a full rework. Accepting this edit!

File: content/games/factions/routines.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:31]: I'm leaning very slightly towards 1. I think we already handled how the third-party non-player "companies" worked in the scenarios for some inspiration. Not covering it here gives us an excuse to chuck it in an open warfare supplement with specific scenarios for larger fights or w/e. Maybe that just belongs in the Eternal Hex which covers a multiplayer campaign where differently controlled companies can intercede.

File: content/games/factions/routines.md [hunk 19]
```diff
@@ -0,0 +1,134 @@
+---
+title: "Routines"
+description: "Rules for non-player Companies in Flagrant Factions"
+weight: 50
+---
+
+These procedures and guidelines apply to Companies not controlled by a player. Unless otherwise
+specified, Routine Companies follow the same rules as other companies. Where decisions are unclear,
+use judgement.
+
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:10]:
```suggestion

{{< hint info >}}
This section assumes that skirmishes are fought between two sides each with a single company. Use your judgement if you're playing with more sides or companies. You've got this.
{{< /hint >}}

```

File: content/games/factions/routines.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (michaeltlombardi) [reply content/games/factions/routines.md:31]: I [took a poke at addressing this in this document](https://github.com/FlagrantGarden/site/pull/52/files#r938106350), once we're happy with the wording we can probably pull it very nearly across. Or this could live only in the core rules or the frontmatter of the text when we're setting the context and just let people understand we meant the _whole_ text.

[the remark this answers is not in the document]
Comment (s-jared) [reply content/games/factions/routines.md:31]: I like this solution. I think it's the shortest path from here to there.

File: content/games/factions/routines.md [hunk 20]
```diff
@@ -0,0 +1,138 @@
+---
+title: "Routines"
+description: "Rules for non-player Companies in Flagrant Factions"
```
Comment (s-jared) [inline content/games/factions/routines.md:3]:
```suggestion
description: "Rules for non-player Companies in Flagrant Factions."
```

File: content/games/factions/routines.md [hunk 21]
```diff
@@ -0,0 +1,138 @@
...
+weight: 50
+---
+
+These procedures and guidelines apply to Companies not controlled by a player. Unless otherwise
+specified, Routine Companies follow the same rules as other companies. Where decisions are unclear,
+use judgement.
+
+{{< hint info >}}
+This section assumes that skirmishes are fought between two sides each with a single company. Use your judgement if you're playing with more sides or companies. You've got this.
```
Comment (s-jared) [inline content/games/factions/routines.md:12]:
```suggestion
This section assumes skirmishes fought between two sides, each with a single company. If you're playing with more sides or more companies, use your judgment. 
```

**Review by s-jared — CHANGES_REQUESTED** [review]

File: content/games/factions/routines.md [hunk 22]
```diff
@@ -0,0 +1,153 @@
...
+weight: 50
+---
+
+These procedures and guidelines apply to Companies not controlled by a player. Unless otherwise
+specified, Routine Companies follow the same rules as other companies. Where decisions are unclear,
+use judgement.
+
+{{< hint info >}}
+This section assumes skirmishes fought between two sides, each with a single company. If you're
+playing with more sides or more companies, use your judgment.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:13]:
```suggestion
playing with more sides or more companies, use your judgment.
```

File: content/games/factions/routines.md [hunk 23]
```diff
@@ -0,0 +1,153 @@
...
+
+## Setup
+
+### Initiative
+
+Incompetent Companies automatically forfeit their initiative roll, counting as a 1 for all purposes.
+When a Routine Company wins the initiative roll-off:
+
+If the Routine Company's Captain has the Shrewd trait, they use it according to their preference:
+- Bold and Ferocious Companies prefer to be the attackers.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:40]:
```suggestion

- Bold and Ferocious Companies prefer to be the attackers.
```

File: content/games/factions/routines.md [hunk 24]
```diff
@@ -0,0 +1,153 @@
...
+friendly Group if possible, or toward the nearest defensible position otherwise.
+
+## Eccentrics
+
+- Groups in Bold and Ferocious Companies with the Defiant trait must always test to activate that
+  trait when possible.
+- Groups in Cautious and Guarded Companies with the Elusive trait must always test to activate when
+  possible.
+- Choose spells randomly unless specified otherwise.
+- Groups with the Casting order may not activate to Cast the same spell twice in a row.
```
Comment (michaeltlombardi) [inline content/games/factions/routines.md:142]:
```suggestion
- Groups with the Casting order may not activate to Cast the same spell twice in a row.
```

File: content/games/factions/routines.md [hunk 25]
```diff
@@ -0,0 +1,158 @@
...
+### Initiative
+
+Incompetent Companies automatically forfeit their initiative roll, counting as a 1 for all purposes.
+When a Routine Company wins the initiative roll-off:
+
+If the Routine Company's Captain has the Shrewd trait, they use it according to their preference:
+
+- Bold and Ferocious Companies prefer to be the attackers.
+- Cautious and Guarded Companies prefer to be the defenders.
+- Arcane Companies prefer whichever role is more eldritch (e.g. the ritualists in the Ritual
```
Comment (s-jared) [inline content/games/factions/routines.md:47]:
```suggestion
- Arcane Companies prefer whichever role is more eldritch (i.e. the ritualists in the Ritual
```

Comment (s-jared) [reply content/games/factions/routines.md:47]: my bad, i used the wrong one.

File: content/games/factions/routines.md [hunk 26]
```diff
@@ -0,0 +1,158 @@
...
+
+1. Line up the Routine Company's Groups from lowest to highest point value along one edge of the
+   table.
+1. To deploy the first Group, roll 1d6:
+   - If the result is _even_, deploy the group farthest right in the line to the nearest deployment
+     zone.
+   - If the result is _odd_, deploy the group farthest left in the line to the farthest deployment
+     zone.
+1. For each remaining Group, roll 1d6:
+   - If the result is odd, select the left-most Group. If even, select the right-most Group.
```
Comment (s-jared) [inline content/games/factions/routines.md:64]:
```suggestion
   - If the result is _odd_, select the left-most Group. If even, select the right-most Group.
```

File: content/games/factions/routines.md [hunk 27]
```diff
@@ -0,0 +1,158 @@
...
+1. Adjust the Company's deployment based on their personality, as needed:
+   - Arcane Companies must deploy Groups with the Casting or Summoning order within 6" of a friendly
+     group, if possible.
+   - Bold Companies must deploy their Groups as close an enemy or objective as possible.
+   - Cautious Companies must deploy their Groups with Shooting, Casting, or Summoning orders within
+     6" of a friendly group and as far away from enemy Groups as possible.
+   - Ferocious Companies must deploy their Groups within range to Attack or Shoot, if possible.
+   - Guarded Companies must deploy their groups within 6" of a friendly Group. If any Groups have
+     the Defensive trait, they must be as evenly dispersed among Group clusters as possible.
+   - Whenever a decision is unclear, choose randomly.
```
Comment (s-jared) [inline content/games/factions/routines.md:78]:
```suggestion
   - Whenever a decision is unclear, choose at random.
```

Comment (s-jared) [reply content/games/factions/routines.md:78]: just to line up with previous language

File: content/games/factions/routines.md [hunk 28]
```diff
@@ -0,0 +1,158 @@
...
+
+- **Arcane:** Give Better Than We Get, Draw First Blood,They Will Know My Name.
+- **Bold:** They Will Reel Before Us, Give Better Than We Get, Draw First Blood, They Will Know My
+  Name.
+- **Cautious:** We Will Not Break, Give Better Than We Get, Rain Hell Upon Them.
+- **Ferocious:** They Will Reel Before Us, We Will Snap Their Spine, Break Even Their Anvil, Draw
+  First Blood, We Will All Shed Blood.
+- **Guarded:** We Will Not Break, Win Without Moving, Do Not Waver, Give Better Than We Get, Rain
+  Hell Upon Them.
+- **Incompetent:** None.
```
Comment (s-jared) [inline content/games/factions/routines.md:94]:
```suggestion
- **Incompetent:** _None_.
```

Comment (s-jared) [reply content/games/factions/routines.md:94]: to technically differentiate from a trait title (so it doesn't look like they should go looking for a trait titled "None").

**Review by s-jared — APPROVED** [review]
